# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/darling/hotfix-fuse.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- darling/src/external/darling-dmg/CMakeLists.txt.vanilla	2023-12-02 11:48:53.187000000 +0100
+++ darling/src/external/darling-dmg/CMakeLists.txt	2023-12-02 11:49:59.189000000 +0100
@@ -118,7 +118,7 @@
 
 	add_definitions(-D_FILE_OFFSET_BITS=64 -DCOMPILE_WITH_LZFSE=1)
 
-	pkg_check_modules(FUSE REQUIRED fuse)
+	pkg_check_modules(FUSE REQUIRED fuse3)
 
 	include_directories(
 		${FUSE_INCLUDE_DIRS}
@@ -126,14 +126,14 @@
 
 	SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -nostdinc")
 
-	wrap_elf(fuse libfuse.so)
+	wrap_elf(fuse3 libfuse3.so)
 
 	add_darling_executable(hdiutil
 		${dmg_sources}
 		src/main-hdiutil.cpp
 	)
 
-	target_link_libraries(hdiutil fuse icucore z bz2 crypto44 xml2 iconv lzfse)
+	target_link_libraries(hdiutil fuse333 icucore z bz2 crypto44 xml2 iconv lzfse)
 	install(TARGETS hdiutil DESTINATION libexec/darling/usr/bin)
 
 endif (NOT DARLING)
